lib/tasks/rh_cloud_inventory.rake

```diff
@@ -2,6 +2,21 @@ require 'tempfile'
 
 namespace :rh_cloud_inventory do
   namespace :report do
+    desc 'Generate inventory report and send it to Red Hat cloud'
+    task generate_upload: [:environment, 'dynflow:client'] do
+      if ! ENV['organization_id'].nil?
+        organizations = [ Organization.where(:id => ENV['organization_id']).first ]
+      else
+        organizations = Organization.unscoped.all
+      end
+
+      User.as_anonymous_admin do
+        organizations.each do |organization|
+          ForemanInventoryUpload::Async::GenerateReportJob.perform_now(ForemanInventoryUpload.generated_reports_folder, organization.id)
```

Comment:
I decided to use that instead of `GenerateAllReportsJob`, since all reports contain:
```ruby
      ensure
        self.class.set(:wait => 24.hours).perform_later
      end
```

As I understand it, calling rake task would automatically schedule next run in 24 hours, effectively adding another upload task to Satellite queue. Using `GenerateReportJob` avoids that problem.